WordPress/Classes/ViewRelated/Comments/CommentModerationBar.swift

```diff
@@ -30,6 +28,23 @@ class CommentModerationBar: UIView {
     private let iPadPaddingMultiplier: CGFloat = 0.33
     private let iPhonePaddingMultiplier: CGFloat = 0.15
 
+    weak var delegate: CommentModerationBarDelegate?
+
+    var comment: Comment? {
+        didSet {
+            currentStatus = CommentStatusType.typeForStatus(comment?.status)
+        }
+    }
```

Comment:
Do we need the Comment object to be passed into this cell? Reading your [comment on this line](https://github.com/wordpress-mobile/WordPress-iOS/pull/17260/files#diff-068cb0f97e51ef29d1181c901df1795395c7808a8302cd41d847fab39a8585cdR212) got me thinking if we could just make the `currentStatus` property public instead, to prevent the Comment object from being mistakenly modified in the future 😄 . What do you think?

Reply:
🤔 you're probably right. I passed the Comment object long before realizing it shouldn't be change here. I'll look at changing that when I add the actual moderation calls, just to make sure.